print_docs.py

```diff
@@ -626,13 +654,36 @@ def write_site_map(partition):
 
 def write_docs_redirect(decl_name, decl_loc):
   url = site_root + decl_loc.url
-  with open_outfile('find/' + decl_name + '/index.html') as out:
+  with open_outfile(f'find/{decl_name}/index.html') as out:
     out.write(f'<meta http-equiv="refresh" content="0;url={url}#{quote(decl_name)}">')
 
 def write_src_redirect(decl_name, decl_loc, file_map):
   url = library_link_from_decl_name(decl_name, decl_loc, file_map)
-  with open_outfile('find/' + decl_name + '/src/index.html') as out:
-    out.write(f'<meta http-equiv="refresh" content="0;url={url}">')
+  with open_outfile(f'find/{decl_name}/src/index.html') as out:
+    out.write(f"""<script src="{site_root}add_commit.js"></script>
+<script>redirectTo("{url}");</script>
+<noscript><meta http-equiv="refresh" content="0;url={url}"></noscript>
```

Comment:
```suggestion
<meta http-equiv="refresh" content="0;url={url}">
```

Does it work without the `<noscript>` tag as well?  `<noscript>` isn't particularly reliable in the age of adblockers.

Reply:
This works, but I'm not sure why. Won't there be some kind of race condition between the javascript and the `<meta>` tag?

Reply:
As a compromise, could we leave the `<noscript>` tag but add a clickable link (`<a>`)?